docs/configuration.rst

```diff
@@ -29,11 +29,9 @@ pyproject.toml
 
 If a file ``pyproject.toml`` exists in the working directory ``pydistcheck`` is run from, ``pydistcheck`` will look there for configuration.
 
-Put configuration in a ``[tool.pydistcheck]`` section, like the following example.
+Put configuration in a ``[tool.pydistcheck]`` section.
 
-.. code-block:: toml
+The example below contains all of the configuration options for ``pydistcheck``, set to their default values.
 
-    [tool.pydistcheck]
-    inspect = False
-    max_allowed_size_compressed = '1G'
-    max_allowed_size_uncompressed = '4.5G'
+.. literalinclude:: ./_static/defaults.toml
+    :language: toml
```

Comment:
what this looks like rendered:

<img width="788" alt="image" src="https://user-images.githubusercontent.com/7608904/209753306-022f61de-1c23-46ed-9328-2f0010c1595b.png">